core/src/main/java/jenkins/util/TimeDuration.java

```diff
@@ -21,27 +21,59 @@ public TimeDuration(long millis) {
         this.millis = millis;
     }
 
+    /**
+     * Returns the duration of this instance in <em>milliseconds</em>.
+     * @deprecated use {@link #getTimeInMillis()} instead.
+     *
+     * This method has always returned a time in milliseconds, when various callers incorrectly assumed seconds.
+     * And this spread through the codebase. So this has been deprecated for clarity in favour of more explicitly named
+     * methods.
+     */
+    @Deprecated
     public int getTime() {
         return (int)millis;
     }
 
+    /**
+     * Returns the duration of this instance in <em>milliseconds</em>.
+     */
     public long getTimeInMillis() {
         return millis;
     }
 
+    /**
+     * Returns the duration of this instance in <em>milliseconds</em>.
```

Comment:
Uh, **seconds**?

Reply:
WAT, how could I miss that, I remember checking, apparently not. Will fix, thanks! :eagle: :eye: